docs/resources/Discovery.md

```diff
@@ -50,6 +50,102 @@ Used to represent a guild's Discovery settings.
 }
 ```
 
+### Discovery Requirements Object
+
+Includes a guild's status on meeting its requirements to be in Discovery.
+
+###### Discovery Requirements Structure
+
+| Field                           | Type                                                                                                                                                      | Description                                                              |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------|
+| guild_id?                       | snowflake                                                                                                                                                 | the guild id                                                             |
```

Comment:
when is it possible for guild_id to not be provided?

Reply:
when the server is verified only sufficient and sufficient_without_grace_period are returned; idk if there are other cases